tests/bud.bats

```diff
@@ -408,6 +408,37 @@ _EOF
   expect_output --substring "Groups:	1000"
 }
 
+@test "build-test skipping unwanted stages with --skip-unused-stages=false and --skip-unused-stages=true" {
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+
+  cat > ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile << _EOF
+FROM alpine
+RUN echo "unwanted stage"
```

Comment:
This isn't unique compared to the message in the third stage, so the test can't check that all of the unwanted stages are still getting processed.